packages/app-config-writer/test/unit/preview-config/__snapshots__/ui5-yaml.test.ts.snap

```diff
@@ -0,0 +1,317 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`update preview middleware config default ui5.yaml w/o index.html 1`] = `
+"# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json
+
+specVersion: \\"3.1\\"
+metadata:
+  name: v4lropconvert0711
+type: application
+server:
+  customMiddleware:
+    - name: fiori-tools-proxy
+      afterMiddleware: compression
+      configuration:
+        ignoreCertError: false # If set to true, certificate errors will be ignored. E.g. self-signed certificates will be accepted
+        ui5:
+          path:
+            - /resources
+            - /test-resources
+          url: https://ui5.sap.com
+        backend:
+          - path: /sap
+            url: https://sap-ux-mock-services-v4-lrop.cfapps.us10.hana.ondemand.com
```

Comment:
do we need this URL for the test? If not I'd replace it with a dummy one.

Reply:
Done in af76444c78a09afb6921c1ed9b4323aa2182321a